### PR TITLE
fix: [ANDROAPP-2846] TEI details title attributes

### PR DESCRIPTION
**Pull request by mmmateos**
[Jira Issue](https://jira.dhis2.org/browse/ANDROAPP-2846)

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt [hunk 1]
```diff
@@ -316,29 +316,22 @@ class EnrollmentActivity : ActivityGlobalAbstract(), EnrollmentView {
...
+                    1 -> text = attrListNotEmpty[0]
+                    else -> text = String.format("%s %s", attrListNotEmpty[0], attrListNotEmpty[1])
+                }
+                setTextColor(Color.WHITE)
             }
-            if(thirdAttr.isEmpty()){
-                binding.teiDataHeader.secundaryAttribute.visibility = View.GONE
+            binding.teiDataHeader.secundaryAttribute.apply {
+                when (attrListNotEmpty.size) {
+                    0, 1, 2 -> visibility = View.GONE
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt]: can you use constants for this magic number ?

Comment (ferdyrod) [reply app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt]: 🤣 "magic number"

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt]: hahahaha I know it sounds funny XD 
https://en.wikipedia.org/wiki/Magic_number_(programming)

Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt]: I don't see them as magic numbers. Just the size of the list... would that fall into the definition?

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt]: ok did not realize about the ".size"
Still, I think its a good practise to put any number/string in constants

**Pull request merged**